ui/panels-plugin/src/plugins/scatterplot/ScatterChart.ts

```diff
@@ -22,5 +22,6 @@ export const ScatterChart: PanelPlugin<ScatterChartOptions> = {
   PanelComponent: ScatterChartPanel,
   // TODO: add a chart options editor plugin, for example:
   // panelOptionsEditorComponents: [{ label: 'Settings', content: ScatterChartOptionsEditorSettings }],
+  supportedQueryTypes: ['TraceQuery'],
```

Comment:
Mentioned this on a sync a while back but I think this panel should be renamed to TraceChart or TraceViewer and ‘scatter’ should be a visual.display option on TimeSeriesChart (similar to the ‘bar’ option)

Reply:
And so scatter would be a visual option on the TraceChart ? This is interesting because it rises the question of preferring to have one chart by query and support multiple visualization, or have one chart by visualization and support multiple queries. 

I'd prefer the latter as sounds more modular but I didn't know that already the TimeSeriesChart was supporting bar display! Is it reusing `BarChart` panel or a complete different implementation.

Reply:
Oh ECharts makes it super easy to switch the series type using their dataset concept, all you do is change between ‘line’, ‘bar’, and ‘scatter’ in that case since they’re all ‘time’ xAxis, basically a one line change plus tweaking their option to adjust styling: 

https://github.com/perses/perses/blob/187d88aa40a261f6d777ce72392222b90accd880/ui/panels-plugin/src/plugins/time-series-chart/utils/data-transform.ts#L148

Our BarChart uses ‘category’ xAxis though so it’s a bit different (more like Datadog’s “Top List” viz)

Reply:
So wayy back TimeSeriesChart was LineChart, if sticking with ScatterChart approach, maybe it should be renamed back, here was that PR #575 and it links to a Matrix chat about it 

Definitely tradeoffs either way, I was one pushing for LineChart back then so it’s funny now that I’m on the other side. Whichever way we go, it should be all-in though and consistent across the core panel kinds

Reply:
Okay I slept on it and what I think is that finally the bar chart as it is right now is even completely timeseries tainted so at the end if we do that the bar chart alone does not make any sense? All of this will be improved anyway when we'll tackle the problem that I exposed on last sync meeting. Summarize it all is in my todo list. 

Waiting for that, Iwill do a separate Pr to align: ScatterChart to TraceChart as you suggested